headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/dto/v1_0/converter/ProductDTOConverter.java

```diff
@@ -294,6 +297,27 @@ private long _getCommerceCatalogId(CPDefinition cpDefinition) {
 		return stream.toArray(RelatedProduct[]::new);
 	}
 
+	private Sku[] _getSkus(
+			CPDefinition cpDefinition, DTOConverterContext dtoConverterContext)
+		throws Exception {
+
+		DTOConverter skuDTOConverter = _dtoConverterRegistry.getDTOConverter(
+			CPInstance.class.getName());
+		List<Sku> skus = new ArrayList<>();
+
+		for (CPInstance cpInstance : cpDefinition.getCPInstances()) {
+			skus.add(
+				(Sku)skuDTOConverter.toDTO(
+					new DefaultDTOConverterContext(
+						dtoConverterContext.getLocale(),
+						cpInstance.getCPInstanceId())));
+		}
+
+		Stream<Sku> stream = skus.stream();
+
+		return stream.toArray(Sku[]::new);
```

Comment:
No need to convert to a stream here.... `skus.toArray(new Sku[0])` would do the same thing. But this is nbd.

Reply:
@ethanbustad I am merging the pull, Please @andrea-ale-sbarra please send a new pr with the change Ethan is asking